tartiflette/schema/transformer.py

```diff
@@ -750,6 +767,127 @@ def parse_directive_definition(
+        name=parse_name(enum_type_extension_node.name, schema),
+        directives=enum_type_extension_node.directives,
+        values=parse_enum_values_definition(
+            enum_type_extension_node.values, schema
+        ),
+    )
+
+    schema.add_extension(enum_extension)
+
+    return enum_extension
+
+
+def parse_input_object_type_extension(
+    input_object_type_extension_node: "InputObjectTypeExtensionNode",
+    schema: "GraphQLSchema",
+) -> "GraphQLInputObjectTypeExtension":
+
+    input_object_extenstion = GraphQLInputObjectTypeExtension(
+        name=parse_name(input_object_type_extension_node.name, schema),
+        directives=input_object_type_extension_node.directives,
+        input_fields=parse_input_fields_definition(
+            input_object_type_extension_node.fields, schema
+        ),
+    )
+
+    schema.add_extension(input_object_extenstion)
+
+    return input_object_extenstion
+
+
+def parse_object_type_extension(
+    object_type_extension_node: "ObjectTypeExtensionNode",
+    schema: "GraphQLSchema",
+) -> "GraphQLObjectTypeExtension":
+
+    object_extension = GraphQLObjectTypeExtension(
+        name=parse_name(object_type_extension_node.name, schema),
+        fields=parse_fields_definition(
+            object_type_extension_node.fields, schema
+        ),
+        directives=object_type_extension_node.directives,
+        interfaces=[
+            x.name.value for x in object_type_extension_node.interfaces
+        ],
+    )
+
+    schema.add_extension(object_extension)
+
+    return object_extension
+
+
+def parse_interface_type_extension(
+    interface_type_extension_node: "InterfaceTypeExtensionNode",
+    schema: "GraphQLSchema",
+) -> "GraphQLInterfaceTypeExtension":
+
+    interface_extension = GraphQLInterfaceTypeExtension(
+        name=parse_name(interface_type_extension_node.name, schema),
+        fields=parse_fields_definition(
+            interface_type_extension_node.fields, schema
+        ),
+        directives=interface_type_extension_node.directives,
+    )
+
+    schema.add_extension(interface_extension)
+
+    return interface_extension
+
+
+def parse_scalar_type_extension(
+    scalar_type_extension_node: "ScalarTypeExtensionNode",
+    schema: "GraphQLSchema",
+) -> "GraphQLScalarTypeExtension":
+
+    scalar_extension = GraphQLScalarTypeExtension(
+        name=parse_name(scalar_type_extension_node.name, schema),
+        directives=scalar_type_extension_node.directives,
+    )
+
+    schema.add_extension(scalar_extension)
+
+    return scalar_extension
+
+
+def parse_union_type_extension(
+    union_type_extension_node: "UnionTypeExtensionNode",
+    schema: "GraphQLSchema",
+) -> "GraphQLUnionTypeExtension":
+
+    union_extension = GraphQLUnionTypeExtension(
+        name=parse_name(union_type_extension_node.name, schema),
+        directives=union_type_extension_node.directives,
+        types=[x.name.value for x in union_type_extension_node.types],
```

Comment:
Shouldn't it be:
```suggestion
        types=parse_union_member_types(union_type_extension_node.types, schema),
```

Reply:
But I only care about the name, then in the bake method of the extension i'll have to do a [x.name.value for x in self.possibleTypes] that's why I took the direct approach.